app/cdap/components/PipelineDiff/store/diffSlice.ts

```diff
@@ -93,6 +98,13 @@ const diffSlice = createSlice({
     },
     modalClosed(state) {
       state.openDiffItem = null;
+      state.focusElement = null;
+    },
+    setFocus(state, action: PayloadAction<string>) {
+      state.focusElement = action.payload;
+    },
+    focused(state) {
```

Comment:
`focused` means `focusElement = null`? The naming vs what it does confuses me a little. Can you explain?